Packs/AzureSecurityCenter/Integrations/AzureSecurityCenter_v2/README.md

```diff
@@ -1,186 +1,107 @@
-   <li><strong>Microsoft Azure Management URL</strong></li>
-   <li><strong>ID (received from the admin consent - see Detailed Instructions (?)</strong></li>
-   <li><strong>Token (received from the admin consent - see Detailed Instructions (?) section)</strong></li>
-   <li><strong>Key (received from the admin consent - see Detailed Instructions (?)</strong></li>
-   <li><strong>Trust any certificate (not secure)</strong></li>
-   <li><strong>Use system proxy settings</strong></li>
-   <li><strong>Default subscription ID to use</strong></li>
-    </ul>
-  </li>
-  <li>
-    Click&nbsp;<strong>Test</strong>&nbsp;to validate the new instance.
-  </li>
-</ol>
-<h2>Commands</h2>
-<p>
-  You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+## Authentication
+
+For more details about the authentication used in this integration, see [Microsoft Integrations - Authentication](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication) .
+
+* After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields. After giving consent, the application has to have a role assigned so it can access the relevant resources per subscription.
+* In order to assign a role to the application after consent was given:
+    * Go to the Azure Portal UI.
+    * Go to Subscriptions, and then Access Control (IAM).
+    * Click Add.
+    * Select a role that includes the following permissions:
+      - Microsoft.Security/locations/read
+      - Microsoft.Security/alerts/read
+      - Microsoft.Security/locations/alerts/read
+      - Microsoft.Storage/storageAccounts/read
+      - Microsoft.Management/managementGroups/read
+      - Microsoft.Security/advancedThreatProtectionSettings/*
+      - Microsoft.Security/informationProtectionPolicies/read
+      - Microsoft.Security/locations/jitNetworkAccessPolicies/*
+      - Microsoft.Security/locations/jitNetworkAccessPolicies/initiate/action
+    * Select the Azure Secruity Center application.
+
+## Configure Azure Security Center v2 on Demisto
+
+1. Navigate to **Settings** > **Integrations**   > **Servers & Services**.
+2. Search for Azure Security Center v2.
+3. Click **Add instance** to create and configure a new integration instance.
+    - **Name**: a textual name for the integration instance.
+    - **Microsoft Azure Management URL**
+    - **ID (received from the admin consent - see Detailed Instructions (?)**
+    - **Token (received from the admin consent - see Detailed Instructions (?) section)**
+    - **Key (received from the admin consent - see Detailed Instructions (?)**
+    - **Trust any certificate (not secure)**
+    - **Use system proxy settings**
+    - **Default subscription ID to use**
+4. Click **Test** to validate the new instance.
+
+
+## Commands
+#### Subscription ID
```

Comment:
@Itay4 Ive added a subscription id section here and added a comment for every command that requires a subscription id.